### DISCUSSION TITLE
405 MethodNotAllowed "Can not remove multiple entries" when patch or delete on service

### DISCUSSION THREAD

**Discussion by HarisHashim** [feathersjs/feathers#3549] [Q&A] [answered]
**How to reproduce**
- Create a new v5 TypeScript, Express & MySQL project using feathers CLI ( `feathers g app` ) with others selected as default.
- Generate authentication and then a service call tasks.
- Do database migration and needed step to populate data
- Use Postmen to authenticate
- Try to delete a users
- Try to delete a tasks
   Both will fail with  code 405
```
{
    "name": "MethodNotAllowed",
    "message": "Can not remove multiple entries",
    "code": 405,
    "className": "method-not-allowed"
}
```

Not sure why it says I am trying to delete multiple entries since it is just deleting one entry. For instance the URL I used in Postman to delete a task is

localhost:3030/tasks?id=1

Also does this means Dove is by default not working for patch and delete? Since so far all the code is generated by CLI. No code change other than what the CLI create. Previously (long time ago), it just work.

TIA
Haris

**Comment by marshallswain** [comment, marked as answer]
Try this URL:

```
localhost:3030/tasks/1
```

https://feathersjs.com/api/client/rest.html#remove
